docs/README.md

```diff
@@ -39,7 +39,17 @@ If you see a project that is under-maintained, submit PRs to fix it as if it was
 
 If you need to contact the nix-community admins, they are listed [here](https://github.com/nix-community/infra#nix-community-administrators).
 
-Come chat with us in [#nix-community](https://matrix.to/#/#nix-community:nixos.org) using the matrix protocol!
+## Support
+
```

Comment:
Could we just have a link to https://github.com/nix-community/infra#support here like we do for the admin list above?

Reply:
I thought about it but it adds friction to people coming on the website. We probably want to consolidate the site, README and also the .github/README at some point.

Reply:
> I thought about it but it adds friction to people coming on the website.

If they want a list of admins they need to click a link anyway so I don't think it's any worse.

Reply:
This is only true for 1/4 of contact methods. It's important that visitors see that info at a first glance and not have to go dig for it.

But also, we probably want to refactor all of this. I will merge this as it's already an improvement and keep iterating on the docs.